viime/views.py

```diff
@@ -213,8 +213,19 @@ def create_csv_file():
 
 @csv_bp.route('/csv/<uuid:csv_id>', methods=['GET'])
 def get_csv_file(csv_id):
-    csv_file = CSVFile.query.get_or_404(csv_id)
-    return jsonify(_serialize_csv_file(csv_file))
+    csv_file = _serialize_csv_file(CSVFile.query.get_or_404(csv_id))
+
+    # inject properties from the validated table model (normalization, transformation, etc.)
+    validated_table = ValidatedMetaboliteTable.query.filter_by(csv_file_id=csv_id).first()
+    if validated_table is not None:
```

Comment:
Perhaps I miscommunicated my issue.  I don't really need these things to come from `/csv/<id>`, but I do need them to come from `/validate`.  Right now, the response from `validate` has these properties, but they're always null regardless of what the user has set.  

The response from `POST /[normalization|transformation|scaling]` contains the correct value echoed back when I change it, but then validate is still null.

Reply:
> normalization, transformation, scaling are all missing from csv/id (expected)

I intended to say that I expected these to be missing, and that behavior was fine.